### DISCUSSION TITLE
golang-random-numbers/

### DISCUSSION THREAD

**Discussion by giscus** [Mr-Destructive/meetgor.com#5] [Q&A]
# golang-random-numbers/

Techstructive Blog - Personal Blog of Meet Gor

https://www.meetgor.com/golang-random-numbers/

**Comment by BanAaron** [comment]
Nice article! I was getting a depreciated warning when using `rand.Seed()` 

This was the only information I could find that clearly explained how to use the new methods

**Reply by Mr-Destructive** [reply]
Glad you found it helpful!

**Comment by techbeamers** [comment]
Good way to demonstrate the random no generation covering the essentials of the topic. I too use to read/write a lot on generating random numbers, even wrote a post on using 10 diff programming languages to generate random numbers at my blog techbeamers.com
